pkg/server/local_observer.go

```diff
@@ -34,6 +34,10 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// DefaultOptions to include in the server. Third-party packages may extend this
+// in their init() function.
+var DefaultOptions []serveroption.Option
```

Comment:
In POC this was done as a work-around to not plumb through cilium, I'm not sure this is a great pattern to keep. Are we able to inject the options instead?